src/client/components/Input/Input.js

```diff
@@ -0,0 +1,11 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import './Input.css';
+
+export default function Input({ placeholder }) {
+  return <input placeholder={placeholder} />;
+}
+
+Input.propTypes = {
+  placeholder: PropTypes.string.isRequired,
```

Comment:
The placeholder should not be required, it should be optional.
You are also missing `value` and `onChange` here. Otherwise, the `Input` component is not usable.